libraries-testing/src/test/java/com/baeldung/dbunit/DataSourceDBUnitTest.java

```diff
@@ -0,0 +1,112 @@
+package com.baeldung.dbunit;
+
+import org.dbunit.Assertion;
+import org.dbunit.DataSourceBasedDBTestCase;
+import org.dbunit.assertion.DiffCollectingFailureHandler;
+import org.dbunit.assertion.Difference;
+import org.dbunit.dataset.IDataSet;
+import org.dbunit.dataset.ITable;
+import org.dbunit.dataset.xml.FlatXmlDataSetBuilder;
+import org.dbunit.operation.DatabaseOperation;
+import org.h2.jdbcx.JdbcDataSource;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.platform.commons.logging.Logger;
+import org.junit.platform.commons.logging.LoggerFactory;
+
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static com.baeldung.dbunit.ConnectionSettings.JDBC_URL;
+import static java.util.stream.Collectors.joining;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DataSourceDBUnitTest extends DataSourceBasedDBTestCase {
+
+        private static final Logger logger = LoggerFactory.getLogger(DataSourceDBUnitTest.class);
+
+        @Override protected javax.sql.DataSource getDataSource() {
```

Comment:
why? let's format it properly, it was ok before. annotations and method definitions on different lines

Reply:
Something went wrong with my autoformatting settings. Let me have a look at what happened.